src/Hydra/Serializer/ErrorNormalizer.php

```diff
@@ -42,6 +42,8 @@ public function normalize($object, $format = null, array $context = [])
         $message = $object->getMessage();
         if ($this->debug) {
             $trace = $object->getTrace();
+        } elseif ($object instanceof FlattenException && isset($context['statusCode']) && $context['statusCode'] >= Response::HTTP_INTERNAL_SERVER_ERROR) {
```

Comment:
According to `Response::isServerError()`:
```php
$context['statusCode'] >= 500 && $context['statusCode'] < 600
```
And IMO if `$context['statusCode']` isn't set, you should use `$exception->getStatusCode()`.